pom.xml

```diff
@@ -49,7 +49,7 @@
             Application Server versions
             (these are downloaded and installed in these versions by Maven for the CI profiles)
          -->
-        <payara.version>6.2021.1-SNAPSHOT</payara.version>
+        <payara.version>6.2022.1.Alpha2</payara.version>
```

Comment:
I believe we should be using snapshot version, so now it would be `6.2022.1.Alpha3-SNAPSHOT`

Reply:
OK. Let me change it